website/code_issues.qmd

```diff
@@ -383,4 +383,44 @@ CRAN doesn't allow negative `warn` options. This setting will turn off all warni
 CRAN recommends using `suppressWarnings()`, which disables warnings only for the specific expression it's applied to, rather than globally.
 
 
+-----------------------------------------
+
+# Using more than 2 Cores
+
+## Problem
+
+You are using more than 2 CPU cores in your examples, vignettes, tests or during installation. 
+
+## Solution
+
+Make sure that you set the maximum number of cores used to 2 in examples, vignettes or tests. 
+During installation, many modern build systems use the `-j` flag. 
+<!-- This was recommended by 'aitap' in a discussion on GitHub -->
```

Comment:
Can you also add @aitap to the contributors section of the [preface.qmd](https://github.com/r-devel/cran-cookbook/blob/main/preface.qmd#L37)?

Reply:
Sure! I will do that 😄